src/api/modal.ts

```diff
@@ -20,11 +20,8 @@ import { ModalWrapper } from '@/types/modal'
  * @doc https://developers.livestorm.co/docs/modal#showiframe
  * 
  */ 
-export async function showIframe(data: { size?: 'normal' | 'large' | 'extraLarge' } & ListenableIframeParams): Promise<ModalWrapper> {
-  const listenableIframe =  await actsAsListenableIframe('modal-show-iframe',
-    { template: data.template, variables: data.variables, onMessage: data.onMessage },
-    { size: data.size }
```

Comment:
C'est normal d'avoir enlevé le param size ?

Reply:
This size option has not been removed. I improved the signature of `actsAsListenableIframe` (https://github.com/livestorm/livestorm-plugin/pull/60/files#diff-099082e5e98a7a9f95f40030dc2e99e29207a7a2c1d1267a3f11f72fe811d606R28)